contracts/deployables/countersign/CountersignV1.sol

```diff
@@ -341,7 +341,13 @@ contract CountersignV1 is
         }
 
         // Check 4: Verify KYC status through external verifier
-        if (!IKYCVerifierV1($.kycVerifier).verify(msg.sender)) {
+        if (
+            !IKYCVerifierV1($.kycVerifier).verify(
+                address(this),
+                msg.sender,
+                verifyingSignature_
+            )
+        ) {
             revert InvalidKYCSignature();
         }
```

Comment:
just thinking here, what if the `verify` function on `KYCVerifierV1` threw the error if the signature isn't valid, instead of duplicating these errors in the calling contracts?

Reply:
I'd prefer for `KYCVerifierV1` to just return true/false. Then the contract calling it have more flexibility. I can imagine situations where if a user isn't KYC verified, we want do some different logic, and not necessarily revert.